src/docs/patterns/user-auth.mdx

```diff
@@ -0,0 +1,296 @@
+    caption={[
+        'Screens shared by the joining-an-organization workflows.',
+        '1: Enter organization code',
+        '2: Enter organization passcode',
+        '3: Success & welcome (awaiting approval)',
+        '4: Success & welcome',
+    ]}
+/>
+
+## Login
+
+Login includes the following variations:
+
+**Basic Login**
+
+A simple login presents fields for the users to enter their credentials and a button to submit them for verification.
+
+<ImageGrid
+    images={[Login]}
+    caption={[
+        'A standard login screen.',
+        '1: Project identifications, such as project name or project logo.',
+        '2: Optionaly, you may add in a simple geometrical background image with a white gradient background.',
+        '3: "Remember me" and "password visibility" are two features that facilitate the user, but expose some security concerns.',
+        '4: Login screens often include extra options, such as "forgot password", "sign up", and "contact your organization admin".',
+    ]}
+/>
+
+**Login through Third Party**
+
+Some applications, especially the consumer-faced ones, are authorized via third-party credentials. Different platforms may have different requirements for this. Here are some commonly used platforms and their instructions:
+
+-   [Login with Amazon](https://developer.amazon.com/docs/login-with-amazon/brand.html)
+-   [Google Sign-in](https://developers.google.com/identity/sign-in/web/sign-in)
+-   [Sign in with Apple](https://developer.apple.com/design/human-interface-guidelines/sign-in-with-apple/overview/introduction/)
+-   [Facebook Login](http://developers.facebook.com/docs/facebook-login/)
+-   [Sign in with Microsoft](https://developer.microsoft.com/en-us/identity/add-sign-in-with-microsoft)
+
+## Reset Password
+
+**Forget password**
+
+The login screen for an application should also present users with a way to recover or reset their password if they have lost or forgotten it. This is typically presented as a hyperlink below the credential fields. Clicking the link will present users with a field to enter their e-mail address. They will receive an email with a hyperlink to reset their password. The workflow involves the following screens:
+
+-   Enter email
+-   Success (email has been sent)
+-   Receive an email for resetting the password
+-   Create password
+-   Success & welcome
+
+<ImageGrid
+    images={[ForgotPasswordEnterEmail, ForgotPasswordConfirmSent, ForgotPasswordNewPassword, ForgotPasswordWelcome]}
+    caption={['1: Enter email', '2: Success (Email has been sent)', '3: Create Password', '4: Success & Welcome']}
+/>
+
+**Change password**
+
+When the users are logged in, they have the option to reset their password by going through the changing password workflow. This workflow can usually be triggered from user preference settings. The workflow involves the following screens:
+
+-   Change password
+-   Success & welcome
```

Comment:
```suggestion
**Change password**

When the users are logged in, they have the option to update or change their password by going through the change password workflow, typically triggered from within the user settings. This workflow includes the following screens:

-   Change password
-   Success
```

Reply:
>the option to update or change their password

what would be the difference between "update" and change" in this context?

Reply:
Effectively no difference. We can just say change.

I think in my head I may have been making a slight distinction between being required to update your password (based on an IT policy) and voluntarily changing your password. But I think change is fine to cover both scenarios.